scripts/exglobal_snow_ensemble_analysis.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# exglobal_snow_ensemble_analysis.py
+# This script creates an SnowEnsAnalysis class
+# and runs the initialize, execute and finalize methods
+# for a global Snow Depth ensemble analysis
+import os
+
+from wxflow import Logger, cast_strdict_as_dtypedict
+from pygfs.task.snowens_analysis import SnowEnsAnalysis
```

Comment:
```suggestion
from pygfs.snowens_analysis import SnowEnsAnalysis
```
I think this should work. 
The others will be updated as well.